datadog_checks_dev/datadog_checks/dev/tooling/commands/dep.py

```diff
@@ -36,6 +36,16 @@
     'cryptography',
     'dnspython',
     'pymysql',  # https://github.com/DataDog/integrations-core/pull/12612
+    'protobuf',  # https://github.com/DataDog/integrations-core/pull/12653
+    'foundationdb',  # https://github.com/DataDog/integrations-core/pull/12653
+    'openstacksdk',  # https://github.com/DataDog/integrations-core/pull/12653
+    'pyasn1',  # https://github.com/DataDog/integrations-core/pull/12653
+    'pycryptodomex',  # https://github.com/DataDog/integrations-core/pull/12653
+    'pysnmp',  # https://github.com/DataDog/integrations-core/pull/12653
+    'clickhouse-driver',  # https://github.com/DataDog/integrations-core/pull/12653
+    'lz4',  # https://github.com/DataDog/integrations-core/pull/12653
+    'pyodbc',  # https://github.com/DataDog/integrations-core/pull/12653
+    'psutil',  # https://github.com/DataDog/integrations-core/pull/12653
```

Comment:
We need a reason for each like in https://github.com/DataDog/integrations-core/pull/12612

Reply:
Context will be added to each Jira card